packages/app/cypress/e2e/top-nav.cy.ts

```diff
@@ -195,7 +195,7 @@ describe('App Top Nav Workflows', () => {
       })
 
       it('hides dropdown when version in header is clicked', () => {
-        cy.findByTestId('cypress-update-popover').findByRole('button', { expanded: false }).as('topNavVersionButton').click()
+        cy.findByTestId('cypress-update-popover').findAllByRole('button').first().as('topNavVersionButton').click()
```

Comment:
These are an example of a test broken by the changes - see PR description for details.